diff --git a/dist/index.js b/dist/index.js
index ccf43d4e9f3c56b62af513f502dcccc9987a4532..59b49aa65620dea06aae6cafd0dce489f4996677 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -5707,11 +5707,11 @@ ${response}`
               message,
               responseMessages,
               state,
-              async (memory) => {
-                return [];
-              }
+              callback
             );
-            if (responseMessages.length) {
+
+            shouldRespond = false;
+            /*if (responseMessages.length) {
               for (const responseMessage of responseMessages) {
                 if (responseMessage.content.providers && responseMessage.content.providers.length > 0) {
                   logger18.debug(
@@ -5723,7 +5723,7 @@ ${response}`
               for (const memory of responseMessages) {
                 await callback(memory.content);
               }
-            }
+            }*/
           }
           await runtime.evaluate(
             message,
