diff --git a/dist/index.js b/dist/index.js
index 3592fc8ae..c075e8ca8 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1695,11 +1695,11 @@ import {
 
 // src/generated/shared/index.ts
 import {
-  AccountRole,
   isProgramDerivedAddress,
   isTransactionSigner as web3JsIsTransactionSigner,
   upgradeRoleToSigner
 } from "@solana/web3.js";
+const AccountRole = { READONLY: 'readonly', WRITABLE: 'writable', SIGNER: 'signer' };
 function expectAddress(value) {
   if (!value) {
     throw new Error("Expected a Address.");
